### PR TITLE
🔥Automatic release via Travis 🔥

### PR DESCRIPTION
**Pull request by karbassi**
Hey @michaelvillar,

I thought I'd make life easier and set up Travis to automatically build and release the app whenever you tag the repo.

The only thing you need to change in this PR is the `api_key`. You can set that up via the `travis` cli.

```
travis setup releases
```

You can see the working version here: https://github.com/karbassi/timer-app/releases/tag/1.x.x

Related to #43

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -74,23 +75,11 @@ fastlane/Preview.html
...
-## compatibility with Xcode 8 and earlier (ignoring not required starting Xcode 9)
-
-## compatibility with Xcode 3 and earlier (ignoring not required starting Xcode 4)
-
-### Xcode Patch ###
-*.xcodeproj/*
-!*.xcodeproj/project.pbxproj
-!*.xcodeproj/xcshareddata/
-!*.xcworkspace/contents.xcworkspacedata
-/*.gcno
```
Comment (michaelvillar) [inline .gitignore]: Why was this removed?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - rm -rf "*.zip" || exit 0
+  - pushd "build/Release"
+  - zip -r "../../Timer.app.zip" "Timer.app/"
+  - popd
+
+deploy:
+  provider: releases
+  skip_cleanup: true
+  api_key:
+    secure: Scw++P6zDI6UJBJeflOWSoQ9ICHjwwPacimCsb29L+PCXFusiiZmb7TTji1mAdaRoFk8qbaHSGVxBcSWJEdIshbI4tNwrcEsewoWz2MVuQSkCfcWAVFNbxc9B5ZtwXsNjig6+VgBxwtkdcb6qI2Yu1ZXGU/T+o6YJV5vB3psmXcISI/oFrflhSsK7QzOc3J+IScVIl/C9hX21gzJhs1GSJdGK8/hLzjuL9NdzOYirqmEuhDdvw1CUNEQOn7fiQMwkerjsy0g/D2frE+UT3Q25pG1TvsvNSDdGYIWO+21X3rTG/YV5MomDp8N1B89gbXBr7opvKNts7p1YAhju0riIu6RyYjw85iF3SDuaBkdDIOUGCEiqRDk3atUnolnQ2lX0/lgPRmH7ojo/ht3wDR0fIXHtkxLo1kp+EOcZuzYxI9cEb37L8i4NPFxXzkTrf1y4rbZiRYHmICmoacdH/oshN/NnuNp0FyKzgFrYTqCUvIXpqnIpPzmAZ6FWU9UvTi0AJHWrtFv4HMAsXllmglclJWzlpFbFhE9hcBpaWvwQuGVodlwI1q6PfTbckyACtNB2emIiQntIfZi6lZbwoetabteciUfYhT7BJ9+K4iIe1KsOuXBLz4laUYLUKyUI7NXO8eJwSOKIZ2Li6UlTyt9rC1h5ieIfpnNr5hk7Sf7a70=
```
Comment (michaelvillar) [inline .travis.yml]: Is this what I need to change with the new api key?

**Comment by michaelvillar** [comment]
That'd be neat! I left a few comments. Thanks!

File: .travis.yml (hunk 2 above)
Comment (karbassi) [reply .travis.yml]: Yes, you're correct. The `api_key` needs to be tied to your github/travis account. 

See PR note on how to create the new key.

File: .gitignore (hunk 1 above)
Comment (karbassi) [reply .gitignore]: For some reason I was having issues with `*.xcshareddata/` folder being committed. I'll revert to test it out.

**Review by karbassi — COMMENTED** [review]
Hope the update fixes things.

**Comment by karbassi** [comment]
@michaelvillar I've made edits.

You can update this branch with your custom encrypted api key so the merge works right away.

Also, before we merge this PR, the deploy must be updated to:

```YAML
deploy:
...
  on:
    tags: true
```

**Comment by karbassi** [comment]
@michaelvillar Any way I can help get this merged?
